app/navigation/stack-param-lists.ts

```diff
@@ -86,10 +86,11 @@ export type MoveMoneyStackParamList = {
 }
 
 export type PhoneValidationStackParamList = {
+  Primary: undefined
```

Comment:
does it means `Primary` can only been `undefined` here? is `Primary` needed then? same question for `welcomePhoneInput` below I guess

Reply:
These types are used with `react-navigation` to define the routes and any params these routes expect -- setting up `Primary: undefined` allows us to do, for example, `navigator.navigate("Primary")` and TS will auto-suggest `"Primary"`. See https://reactnavigation.org/docs/typescript#type-checking-the-navigator